tests/ui/src/utilities/globalSetup.ts

```diff
@@ -12,9 +12,13 @@ async function globalSetup() {
     const workspace = process.env.WORKSPACE as string;
     const env = process.env.ENV as string;
     const region = process.env.REGION as string;
-    const distribution = getCloudFrontUrl(region, env, workspace);
-    const baseUrl = JSON.parse(distribution).DomainName;
-    process.env.baseUrl = baseUrl;
+    const distribution = await getCloudFrontUrl(region, env, workspace);
+    if (distribution) {
+      const baseUrl = distribution.DomainName;
+      process.env.baseUrl = baseUrl;
+    } else {
+      console.log('No domain name found');
+    }
```

Comment:
If you directly return the domain name, you can simplify this a good bit.

```suggestion
    process.env.baseUrl = await getCloudFrontUrl(region, env, workspace);
    if (!process.env.baseUrl) {
      console.log('No domain name found');
    }
```